sdk/keyvault/keyvault-secrets/tests.yml

```diff
@@ -7,8 +7,3 @@ extends:
       ServiceDirectory: keyvault
       TimeoutInMinutes: 59
       SupportedClouds: 'Public,UsGov,China'
-      AdditionalMatrixConfigs:
```

Comment:
I trust that this change was necessary, but curious what it's for?

Reply:
TL;DR: this is how we generated the multi-service-version CI runs (7.0, 7.1, etc)

You know how our CI matrix generates N jobs (node18windows, node22mac, etc)? This allows passing in additional configs you want run. In our case it's the usual workloads + one per older service version (defined in sdk/keyvault/keyvault-secrets/platform-matrix.json)

As we decided to abandon that effort due to low adoption, removing the extra CI runs that are no longer necessary. LMK if any of that is unclear 👍

Reply:
Ah, I forgot we were removing that -- makes sense. Thanks!